src/engraving/libmscore/note.cpp

```diff
@@ -2009,11 +2009,6 @@ Element* Note::drop(EditData& data)
         n->setTpc2(Ms::transposeTpc(n->tpc1(), v, true));
         // replace this note with new note
         n->setParent(ch);
-        if (this->tieBack()) {
-            n->setTieBack(this->tieBack());
-            n->tieBack()->setEndNote(n);
-            this->setTieBack(nullptr);
-        }
```

Comment:
I'm confused now, is that the chage you really wanted?

Reply:
If so: this has not been part of my (meanwhile merged) PR

Reply:
It is however partt of your #8825, and as such in the current code base. Should it get removed again, ir is this just another tell-tale sign of a botched local repo?